torchdata/datapipes/iter/load/README.md

```diff
@@ -12,10 +12,6 @@ S3 IO datapipes are included when building with flag `BUILD_S3=1`. The following
 source with S3 datapipes.
 
 ```bash
-pip uninstall torchdata -y
-git clone https://github.com/pytorch/data.git
-cd data
-python setup.py clean
 BUILD_S3=1 python setup.py install
```

Comment:
No change needed. Just a question, I assume this isn't relevant if people install from `pip` or `conda`?

Reply:
Users don't need to uninstall `torchdata` and re-install with `BUILD_S3`. I will add a few words about torchdata releases with pre-assembled AWSSDK.